example-widget-semantic-ui/src/PowerLevelsPage/PowerLevelsPage.test.tsx

```diff
@@ -109,28 +109,28 @@ describe('<PowerLevelsPage />', () => {
     const listbox = await screen.findByRole('listbox', { expanded: false });
 
     expect(
-      within(listbox).getByRole('option', {
+      await within(listbox).findByRole('option', {
```

Comment:
This should be changed back to `getByRole` We know it is here because it was checked some lines above. Generally said we should use async only as much as needed.

Reply:
rolling this back leads to `TestingLibraryElementError: Unable to find an accessible element with the role "option" and name "@another-user"` and

<img width="593" alt="Screenshot 2024-05-03 at 12 23 11" src="https://github.com/nordeck/matrix-widget-toolkit/assets/26399/7890d501-ba39-4ad5-ae9c-c08277930644">

Reply:
:+1: I did not see that one waits for the list and the other for the option :facepalm: